Chapter10/pretty_print.erl

```diff
@@ -0,0 +1,194 @@
+-module(pretty_print).
+-export([index/0, index/1, accumulate/1, prettyList/1, pad/2]).
+-include("usr.hrl").
+
+-vsn(1.0).
+
+
+get_path() -> "/Users/daniel.harris/Documents/GitErlang/learning-erlang/Chapter10/indexme.txt".
+
+index() -> index(get_path()).
+
+
+% Exercise 10-2: Indexing Revisited
```

Comment:
can you clarify what you did exactly for exercise 10-2? Unfortunately I don't see anything.

Reply:
I added "{keypos, 1}" to make sure the set is keyed on the first element of "{Word, LNs}", if I've done it wrong let me know and I'll update the code

Reply:
So, book says that:
>  ETS table is an ordered set keyed on the Word field of tuples of the form __{Word, LNs}, where LNs is a list of line numbers on which the word occurs__

In your example code I see that ets consists of `{Word, N}` entries. I assume that exercise asks you to change the code so that ets table has `{Word, LNs}` entries. Currently `prettyIndex/0` goes through all entries in the table and find all word's lines and then pretty print it.

Reply:
All fixed, the ets now consists of {Word, LN} instead of {Word, N}